### ISSUE TITLE
[BotConfiguration.bot ] file missing in QnA sample

### ISSUE THREAD

**Issue by JohnMasen** [microsoft/BotBuilder-Samples#739]
the file 
[https://github.com/Microsoft/BotBuilder-Samples/blob/master/samples/csharp_dotnetcore/11.qnamaker/BotConfiguration.bot] is missing

**Comment by 3bu1** [comment]
same issue, i have manually added .bot file, now it says `TypeError: Cannot read property 'kbId' of null`

**Comment by ryanvolum** [comment]
This is true of several node samples as well (including the node qnaMaker sample)

**Comment by deepak2016** [comment]
I'm facing the same issue and due to this I'm not able to build the project and make progress.

**Comment by jasonsargood** [comment]
Just downloaded samples and tried the NLP with Luis and again no BotConfiguration.bot file

**Comment by sgellock** [comment]
We had PR #910 out for review to fix this, but we pulled it in order to fix some bot file naming inconsistencies.  You can reference that PR for what's necessary to get yourself unblocked until we get the new PR merged.  I'd like to get this all wrapped up today if possible
